### PR TITLE
[New Rule] Privilege Escalation via Windir Environment Variable

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/637

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
image

File: rules/windows/privilege_escalation_rogue_windir_environment_var.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+creation_date = "2020/11/26"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a privilege escalation attempt via rogue Windows Directory (Windir) Environment Variable. This is a known
+primitive that is often combined with other vulnerabilities to elevate privileges.
```
Comment (bm11100) [inline rules/windows/privilege_escalation_rogue_windir_environment_var.toml:11]:
```suggestion
Identifies a privilege escalation attempt via a rogue Windows directory (Windir) environment variable. This is a known
primitive that is often combined with other vulnerabilities to elevate privileges.
```

File: rules/windows/privilege_escalation_rogue_windir_environment_var.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+name = "Privilege Escalation via Windir Environment Variable"
+references = ["https://www.tiraniddo.dev/2017/05/exploiting-environment-variables-in.html"]
+risk_score = 71
+rule_id = "d563aaba-2e72-462b-8658-3e5ea22db3a6"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+registry where registry.path : ("HKEY_USERS\\*\\Environment\\windir", "HKEY_USERS\\*\\Environment\\systemroot") and
```
Comment (bm11100) [inline rules/windows/privilege_escalation_rogue_windir_environment_var.toml:25]: looks good, just waiting to have syntax validate properly in a dev tools instance before approving

Comment (bm11100) [reply rules/windows/privilege_escalation_rogue_windir_environment_var.toml:25]: confirmed logic

**Review by bm11100 — APPROVED** [review]
lgtm

**Review by devonakerr — APPROVED** [review]

**Pull request merged**